rolemanager/lib/Controllers/login_controller.dart

```diff
@@ -1 +1,48 @@
+import 'package:flutter/material.dart';
 
+class LoginController {
+  static bool isNullValidator(
+      TextEditingController _controller, bool _validator) {
+    if (_controller.text.isEmpty) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  static bool isEmailValid(TextEditingController _controller, bool _validator) {
+    final emailRegExp = RegExp(r"^[a-zA-Z0-9.]+@[a-zA-Z0-9]+\.[a-zA-Z]+");
```

Comment:
ENG:
Change regex to accept underline "**_**" in email case 

PT: 
Mudar o regex do email pra aceitar underline "**_**" também